docs/standard/analyzers/portability-analyzer.md

```diff
@@ -7,35 +7,58 @@ ms.assetid: 0375250f-5704-4993-a6d5-e21c499cea1e
+
+### View and interpret portability result
+Only APIs that are unsupported by a Target Platform appear in the report. 
+After running the analysis in Visual Studio, you'll see your .NET Portability report pops up. 
 
 ![Portability Report](./media/portability-analyzer/portability-report.png)
 
-If you don’t want to use Visual Studio, you can use the Portability Analyzer from the command prompt. Just download the API Portability Analyzer from the [Microsoft/dotnet-apiport](https://github.com/Microsoft/dotnet-apiport/releases) repository.
+If you used [ApiPort console app](https://github.com/Microsoft/dotnet-apiport/releases), your .NET Portability report is saved as a file in your specified format, default is in an Excel file (*.xlsx*) in your current directory. 
 
-* Type the following command to analyze the current directory: `\...\ApiPort.exe analyze -f .`
-* To analyze a specific list of .dll files, type the following command: `\...\ApiPort.exe analyze -f first.dll -f second.dll -f third.dll`
+#### Portability Summary 
+This session of the report gives the assembly level overall view of the portability percentage.  As in above report example, it means that 89.74% of .NET Framework APIs the ConsoleAppFramework app used are available in .NET Core + Platform Extensions v2.2. If you run .NET Portability Analyzer tool against multiple assemblies, each of the assembly should have a row in the Portability Summary report.
+
+#### Details
+List the APIs missing from one of the Target Platforms. 
+Target type: the type has missing API from a Target Platform
+Target member: the method is missing from a Target Platform
+Header for assembly name entries: the .NET Framework assembly that the missing API lives in.
+Each of the selected Target Platforms is one column, such as ".NET Core": "Not supported" value means the API is not supported on this Target Platform.
+Recommended Changes: recommended API or technology to change to. Currently, this field is empty or out of date for a lot of APIs. Due to the large number of APIs, we have big challenge to keep this up. We are looking at alternate solutions to provide helpful information to customer.
+
+In Visual Studio,  under the **Messages** tab in the **Error List**, it displays missing APIs with caller file and line number. You can jump to problem areas directly from the **Messages** tab.
 
-Your .NET Portability Report is saved as an Excel file (*.xlsx*) in your current directory. The **Details** tab in the Excel Workbook contains more information.
+#### Missing assemblies
+You may have a list of the assemblies in "Header for assembly name entries" column. It normally are the assemblies that your analyzed assembly reference to and they are not .NET Framework assemblies. If it's an assembly that you own, include it in the portability analyze run. If it's third party library, looks for if they have newer version supporting .NET Core or .NET Standard. If so, consider moving to the newer version.  
```

Comment:
Where's that column? Do we need to run this against a better sample?

There are some typos here (It normally are, your analyzed assembly reference to, portability analyze run, looks for). Do you want to review this and I can propose some changes after I see your intention.

Reply:
@mairaw  Yes, we need update the sample image. Where should I upload the new sample images?

Reply:
Great idea! Updated to use https://aka.ms/apiportdownload for console app download.